devicetypes/smartthings/zigbee-battery-accessory-dimmer.src/zigbee-battery-accessory-dimmer.groovy

```diff
@@ -143,16 +192,28 @@ def setLevel(value, rate = null) {
 }
 
 def ping() {
-	zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021)
+	if(isCentraliteSwitch()) {
+		zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0020)
+	} else {
+		zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021)
+	}
 }
 
 def installed() {
-	sendEvent(name: "switch", value: "on", displayed: false)
-	sendEvent(name: "level", value: 100, displayed: false)
+	sendEvent(name: "switch", value: "on")
+	sendEvent(name: "level", value: 100)
 }
 
 def configure() {
 	sendEvent(name: "checkInterval", value: 2 * 60 * 60 + 10 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
-	zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021) + zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021, DataType.UINT8, 30, 10 * 60, null)
+
+	if(isCentraliteSwitch()) {
+		zigbee.addBinding(zigbee.ONOFF_CLUSTER) + zigbee.addBinding(zigbee.LEVEL_CONTROL_CLUSTER) +
+				zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x20) +
```

Comment:
just checking: the device doesn't support attribute 0x21 here?

Reply:
it's supports, but only you can read value from 0x21. When we did report config for 0x21 device was not sending reports, so we decided to read values and set configuration reporting for voltage reading.